config/environments/development.rb

```diff
@@ -7,7 +7,7 @@
 
   config.consider_all_requests_local = true
 
-  if Rails.root.join("tmp", "caching-dev.txt").exist?
+  if Rails.root.join("path/to").exist?
```

Comment:
Ignoring this for now, because I am getting conflicting messages from Hound and Rubocop about `Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.`